### ISSUE TITLE
Use series_synonym if series_title isn't found

### ISSUE THREAD

**Issue by glaubermmferreira** [edhaker13/myanimelist#9]
I've been wondering if it's possible to use *series_synonym* for search if flexget doesn't find anything using *series_title*. I want this because sometimes MAL uses incorrect/alternate titles for some animes, Gakkou Gurashi! is one example.

Also, thanks for your plugin.

**Comment by edhaker13** [comment]
Thanks, I appreciate it.

This is a good idea, and I think I actually did this in my development
version I just haven't cleaned it up for a commit.

I'll try to commit it to master when I get back home, I think I never saved
it outside of where I'm using it.

In the meantime, you can redefine the series after import.

``` yml
#use plugin
configure_series:
  from:
    myanymelist:
      username: 'your username'
#add additional names
series:
  - Gakkou gurashi:
      alternate_name: Gakkougurashii!
```

**Comment by edhaker13** [comment]
Implemented in e61e10e1a1396b5ef75d7934c6718ba15750b2bc.

Known issues: 
Second seasons may have the same title as alternate names, easiest solution is to remove previous season from database.

Even with alternate titles groups like horriblesubs use completely different name (Owari no Seraph: Nagoya - Seraph of the End S2), you can still add an name as mentioned above.

Shows may change spelling mid way: you either remove the old name and start again or add it as an alternate name.
